source/includes/mongosync-and-OIDC.rst

```diff
@@ -0,0 +1,4 @@
+Starting in ``mongosync`` 1.9.0, you can use ``mongosync`` to connect to
+MongoDB clusters with OpenID Connect (OIDC). For example, you can
+connect to clusters running on Microsoft Azure and Google Cloud
+Platform.
```

Comment:
(short description)
Please add a sentence or two explaining what OIDC is and why you'd use it.